js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+
+$emailInput.addEventListener('focusout', function (e) {
+  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+
+  defaultStyle();
```

Comment:
함수는 어떤 동작을 하는지 `행위`에 대해서 함수명에 들어가 있는게 좋은데요!
아래와 같은 느낌도 좋을 것 같아요!

```suggestion
  setDefaultStyle(); // 또는 resetStyle();
```